bootstrap/modules/external-secrets/iam.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_iam_access_key" "externalsecrets" {
+  user    = aws_iam_user.externalsecrets.name
+}
+
+#tfsec:ignore:aws-iam-no-user-attached-policies
+resource "aws_iam_user" "externalsecrets" {
+  name = "${var.cluster_name}_cluster_secrets_user"
+  path = "/externalsecrets/"
+}
+
+#tfsec:ignore:aws-iam-no-policy-wildcards
+resource "aws_iam_role_policy" "externalsecrets" {
+  name = "${var.cluster_name}_cluster_ssm_policy"
+  role = aws_iam_role.assume_role.id
+
+  # Terraform's "jsonencode" function converts a
+  # Terraform expression result to valid JSON syntax.
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "secretsmanager:GetResourcePolicy",
+        "secretsmanager:GetSecretValue",
+        "secretsmanager:DescribeSecret",
+        "secretsmanager:ListSecretVersionIds"
+      ],
+      "Resource": [
+        "arn:aws:secretsmanager:eu-central-1:571075516563:secret:kaas/k8s/shared-secrets/*"
+      ]
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role" "assume_role" {
+  name = "ExternalSecretsUser"
```

Comment:
If we deploy 2 clusters; there will be 2 roles with the same name, this won't work.
adapt to 
`name = "${var.cluster_name}_cluster_secrets_user"` like we used in the `aws_iam_user`